src/docfx/lib/moniker/Moniker.cs

```diff
@@ -6,18 +6,18 @@
 
 namespace Microsoft.Docs.Build
 {
-    internal class Moniker
```

Comment:
>Moniker [](start = 19, length = 7)

Personally prefer the Moniker.Name, Moniker.Product

Reply:
agree！


---
In reply to: [230629372](https://github.com/dotnet/docfx/pull/3575#discussion_r230629372) [](ancestors = 230629372)